src/sensitivity.jl

```diff
@@ -54,3 +54,26 @@ function plotUnitSensitivity(paramRange, result, realParam, i)
     plot!([realParam], seriestype = "vline", margin = 0.3cm, legend = :false)
     ylims!((1E2, 1E5))
 end
+
+""" This function calculates the simulated cell number for a pair of drugs in their exact concentration.  """
+function calc_cellNum(pDr1, pDr2, g0)
+    g1d1, g2d1, _ = predict(pDr1, g0, 96.0)
+    g1d2, g2d2, _ = predict(pDr2, g0, 96.0)
+    normNum = 1.0 .- [(g1d1 + g2d1)/g0, (g1d2 + g2d2)/g0]
+    combin = -((normNum[1] + normNum[2] - normNum[1] * normNum[2]) .- 1.0) * g0 
+    return combin
+end
+
+""" Calculates the difference between the bliss_cell number and bliss_params. """
+function calc_diff(Dr1_params, Dr2_params, effs, combin, concs, conc1_indx, conc2_indx, g0)
+    bliss_comb = DrugResponseModel.fullCombinationParam(Dr1_params, Dr2_params, effs, 8, conc1_indx, conc2_indx)
+    bliss_comb_cellnum = BlissModelComb(bliss_comb, g0)[conc1_indx, conc2_indx]
+    return combin - bliss_comb_cellnum
+end
+
+function get_derivative(x, Dr2_params, effs, combin, concs, conc1_indx, conc2_indx, g0)
+    # "k" is the index of the parameter we want to calculate the derivative of diff with respect to it.
+    fd(x) = calc_diff(x, combin, concs, Dr1_indx, Dr2_indx, g0) # closure?
+    out = DiffResults.DiffResult(x)
+    return ForwardDiff.derivative.(out, fd, x)
+end
```

Comment:
@aarmey I couldn't get around the error regarding how to call the `ForwardDiff.derivative!(...)`. I couldn't find any relevant examples online. Could you please help me with this function?